lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -325,7 +323,7 @@ private JourneyResponse getJourneyResponse(
                             auditEventUser,
                             deviceInformation,
                             contraIndicators)
-                    : buildNoMatchResponse(contraIndicators, preferredNewIdentityLevel);
+                    : buildNoMatchResponse(contraIndicators, lowestGpg45ConfidenceRequested);
```

Comment:
Could buildNoMatchResponse have the parameter defined be renamed to lowestGpg45ConfidenceRequested

Reply:
In `buildNoMatchResponse` the parameter is called `preferredNewIdentityLevel` which seems to match what it represents in that function quite well?